Gruntfile.js

```diff
@@ -131,7 +131,7 @@ module.exports = function(grunt) {
                 dest: 'build/raven.test.js',
                 options: {
                 browserifyOptions: {
-                    debug: true // source maps
+                    debug: false// source maps
```

Comment:
Source map generation for tests is/has been broken so I just disabled it to make my life easier debugging this. I'll revisit in future.